.github/workflows/compress-images.yml

```diff
@@ -0,0 +1,28 @@
+name: Compress Images
+on:
+  pull_request:
+    # Run Image Actions when JPG, JPEG, PNG or WebP files are added or changed.
+    # See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths for reference.
+    paths:
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.webp'
+jobs:
+  compress-images:
+    # Only run on Pull Requests within the same repository, and not from forks.
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Compress Images
+        uses: calibreapp/image-actions@main
+        with:
+          jpegQuality: '70'
+          jpegProgressive: false
+          pngQuality: '70'
+          webpQuality: '70'
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Maybe bump these to like 85 or something? Don't want to crush them

Reply:
Checkout the diffs! 70 is not too low at all 

https://github.com/spicetify/spicetify-themes/pull/766/commits/f711cb314f22ad51c4928cab43a1534aa9ae1b33#diff-3c4454a0af8854a1ece53111c2fb0d25

Reply:
Ah, nice